gnc/navigator_controller/src/mrac_controller.py

```diff
@@ -22,14 +93,15 @@ class MRAC_Controller:
 
     def __init__(self):
         '''
-        ###Explain what is assigned here
+        Set hard-coded tunable parameters (gains and desired vehicle speed limit).
+        The boat model section's values are used 
```

Comment:
Oops, that's embarrassing.. not sure where the rest of that sentence went
I'll touch that up with an inevitable PR at the next lake day.